apps/vax/mix.exs

```diff
@@ -0,0 +1,59 @@
+defmodule Vax.MixProject do
+  use Mix.Project
+
+  def project do
+    [
+      app: :vax,
+      version: "0.1.0",
+      elixir: "~> 1.7",
+      start_permanent: Mix.env() == :prod,
+      deps: deps(Mix.env()),
+      package: package(),
+      test_paths: ["test"]
+    ] ++ docs()
+  end
+
+  # Run "mix help compile.app" to learn about applications.
+  def application do
+    [
+      extra_applications: [:logger]
+    ]
+  end
+
+  # Run "mix help deps" to learn about dependencies.
+  defp deps(:publish) do
+    [
+      {:ecto, "~> 3.7"},
+      {:nimble_pool, "~> 0.2.6"}
```

Comment:
I suggest we add TODO or FIXME here, given that today if we publish this to hex the code would not work.
One way of fixing that properly afaik to add dependency with `sparse` option that will point to this git repository